### DISCUSSION TITLE
Welcome to thirdweb-art-engine Discussions!

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/thirdweb-art-engine#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PhantomPirateCrew** [comment]
Add a provenance hash code for additional public records. See NFT Chef coding for details. 
![Screenshot_6](https://user-images.githubusercontent.com/106455928/224883028-67b5c24a-8a97-4d5d-bd49-71af4a45abea.png)

**Comment by PhantomPirateCrew** [comment]
Another NFT Chef code is the nesting folders. Best use case is with different characters and being able to generate them all in one batch. 
<img width="1930" alt="Branching with nested folders" src="https://user-images.githubusercontent.com/106455928/224883651-c365f581-1943-4c1f-b467-e1a63f4f25d0.png">

**Comment by PhantomPirateCrew** [comment]
Being able to start off with 100 for testing purposes then immediately scaling to a higher amount without regenerating images would also be nice! This is from GitHub (datlips_art_engine)
![Screenshot_8](https://user-images.githubusercontent.com/106455928/224884298-dc926131-6409-421e-a99e-c41141930d28.png)

**Comment by PhantomPirateCrew** [comment]
Preview_gif is a good feature too for being able to look at all NFTs to ensure quality control.

**Comment by PhantomPirateCrew** [comment]
Flagging incompatible layers and forced layer combinations for items the nesting structure can't solve. (NFTChef)
![Screenshot_9](https://user-images.githubusercontent.com/106455928/224885238-3b560bbc-331c-45da-a1c5-e4b3fb2c0b2e.png)

**Comment by PhantomPirateCrew** [comment]
This one is a lifesaver if you have a trait that needs to be in the images, but not the metadata. Removing the trait completely really helped me in my collection, so I didn't have to manually delete anything. (NFTChef)
![Screenshot_10](https://user-images.githubusercontent.com/106455928/224885535-0f4b4400-d8ab-4db5-a5be-e3bdeca907e8.png)

**Comment by PhantomPirateCrew** [comment]
Forgot to introduce myself, I am a newbie developer learning web3 to launch my own NFT collection. I'm in the Thirdweb discord as "Hallyzee123". I've been learning blockchain and crypto from a developer standpoint since April of last year, but been in crypto for a few years prior. I look forward to collaborating!
